### ISSUE TITLE
How can achieve this format on exporting to CSV 

### ISSUE THREAD

**Issue by hallowcard13** [SkalskiP/make-sense#201]
` TRAIN,gs://folder/image1.png,car,0.1,0.1,,,0.3,0.3,,`

I following this tutorial on https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/tutorials/model_maker_object_detection.ipynb#scrollTo=xushUyZXqP59

and i want to re train the model with my own dataset. I tried exporting it to csv, But when i run the code to train the model i got  an error `AttributeError: 'NoneType' object has no attribute 'label_map'` 

Can you please advise. Thank you

**Comment by SkalskiP** [comment]
Hello @hallowcard13 👋. Is this a problem with make-sense?

**Comment by hallowcard13** [comment]
> Hello @hallowcard13 👋. Is this a problem with make-sense?

No it's not from make-sense app. I was trying to ask if there is a chance we can do the same csv format on that tutorial provided by google

**Comment by SkalskiP** [comment]
@hallowcard13 could that format be AutoML format? If so tan we already have issue about that https://github.com/SkalskiP/make-sense/issues/80

**Comment by hallowcard13** [comment]
> @hallowcard13 could that format be AutoML format? If so tan we already have issue about that #80

yes it is, Saw and read about the issue. I can help you, i know javascript and react but 0 on typescript hehe. If you could point me into which class or file is responsible for creating the csv and vertices that would speed up the process and for me to learn about typescript. Looking forward thank you

**Comment by SkalskiP** [comment]
Great! Happy to see you would like to give me a helping hand 🙋 ! The file that you are looking for is `make-sense/src/logic/export/RectLabelsExporter.ts`. The method that parses rectangle labels to CSV is `RectLabelsExporter.exportAsCSV`. It would be also great if you could update our docs and provide some info on that file format. Simple update of that `make-sense/docs/export.md` file will be enough.

**Comment by hallowcard13** [comment]
> Great! Happy to see you would like to give me a helping hand 🙋 ! The file that you are looking for is `make-sense/src/logic/export/RectLabelsExporter.ts`. The method that parses rectangle labels to CSV is `RectLabelsExporter.exportAsCSV`. It would be also great if you could update our docs and provide some info on that file format. Simple update of that `make-sense/docs/export.md` file will be enough.

Thank you for the information i will find sometime this weekend to start working on it. I'll keep you posted on PR

**Comment by SkalskiP** [comment]
@hallowcard13 this is great! I'm so happy to hear that ;)

**Comment by wwfish** [comment]
Just dropping into figure out if we did manage to get the CSV to export for a float between 0 and 1 as used in ML Vision CSV data import?

Anybody got a work around adapting the pixel coordinates to ML formats between 0 and 1?
